Neos.Utility.Lock/Classes/TYPO3/Flow/Utility/Lock/Lock.php

```diff
@@ -78,6 +77,18 @@ public static function setLockManager(LockManager $lockManager = null)
 
     /**
      * Releases the lock
+     * @param boolean $exclusiveLock TRUE to, acquire an exclusive (write) lock, FALSE for a shared (read) lock. An exclusive lock is the default.
+     * @return void
+     */
+    public function acquire($exclusiveLock)
```

Comment:
Arf ... we acquire the lock in the constructor and in the aquire method ... that's bad :(